.github/workflows/test-windows.yml

```
@@ -26,10 +26,13 @@ jobs:
             gpu-arch-type: cuda
             gpu-arch-version: "11.7"
       fail-fast: false
-    uses: pytorch/test-infra/.github/workflows/windows_job.yml@main
+    uses: pytorch/test-infra/.github/workflows/windows_job.yml@win-gpu
     with:
+      test-infra-ref: win-gpu
```

After pytorch/test-infra#4104 is merged, we need to revert this

```suggestion
    uses: pytorch/test-infra/.github/workflows/windows_job.yml@main
    with:
```